### PR TITLE
feat:[JIRA:HCPSDKFIORIUIKIT-2180] Chart Color updates

### PR DESCRIPTION
**Pull request by KevinZK**

### CODE REVIEW

File: Sources/FioriThemeManager/Colors/ColorStyle.swift [hunk 1]
```diff
@@ -862,58 +862,58 @@ public enum ColorStyle: String, CaseIterable {
     // MARK: - Chart Colors (13 Styles)
     
     /// Chart 1
-    /// (dark variant) ![](https://dummyimage.com/48x18/5899DA.png&text=+)          Hex value: 5899DA
-    /// (light variant) ![](https://dummyimage.com/48x18/74B3F0.png&text=+)          Hex value: 74B3F0
+    /// (dark variant) ![](https://dummyimage.com/48x18/4DB1FF.png&text=+)          Hex value: 4DB1FF
```
Comment (billzhou0223) [inline Sources/FioriThemeManager/Colors/ColorStyle.swift:865]: The mapping is wrong. `5899DA` should be mapped to `1B90FF`.

**Review by billzhou0223 — CHANGES_REQUESTED** [review]
Need to double check and fix all the mapping issues.

**Comment by billzhou0223** [comment]
You need to sign the CLA first.

**Comment by dyongxu** [comment]
@KevinZK Your new colors did not replace the correct value based on the mode (light or dark).   Currently the order of "light / dark" color is reversed.
To avoid this, e.g. you can simply replace the value of E8743B to E76500 for chart2, so it won't go to the wrong location.

**Review by billzhou0223 — APPROVED** [review]

**Pull request merged**